Gopkg.toml

```diff
@@ -1,25 +1,3 @@
-
-# Gopkg.toml example
```

Comment:
The example is quite useful to leave in, it is not doing any harm and helps self-document.

Reply:
restored